node/src/components/deploy_acceptor.rs

```diff
@@ -760,12 +798,12 @@ impl DeployAcceptor {
                         }
                         None => {
                             debug!(?contract_version, "invalid contract at version");
```

Comment:
Not really required but it would be nice to also update all `debug` logs to print out the error info since it contains much more information.

Reply:
As discussed offline, the errors don't actually give any additional info over the logs, so I'll leave this as is.